src/behavior.scm

```diff
@@ -203,6 +203,29 @@
 			(ListLink (Node "--- Glance at new person")))
 	))
 
+;; New recognized person sequence
+(DefineLink
+	(DefinedPredicate "Interacting Sequence for recognized person")
+	(SequentialAnd
+		;(True (DefinedSchema "glance at new person"))
+		(True (Put ; Setting interaction target for "look at person"
+			(DefinedPredicate "Set interaction target")
+			(RandomChoice (Put; FIXME Replace by multiple face tracking.
+				(DefinedSchemaNode "Get recognized face's face id")
+				(DefinedSchema "Get recognized faces")))))
+		(DefinedPredicate "look at person")
+		(True (Put
+			(DefinedPredicate "Greet recognized person")
```

Comment:
By "short-circuiting nlp", I don't mean "sureal", I mean "knowing that she just said hello, so that if you ask her, what did you just say/do, she could answer that correctly".  That means that her recent actions need to be recorded as a part of her self-model, along with recent changes to the world-model as perceived by her senses.   Again, the code in chatbot-eva tried to do a rudimentary version of that; it needs refinement and expansion, and I need to provide more info on how that might be accomplished.